multi_partner_learning.py

```diff
@@ -38,7 +40,10 @@ def __init__(self,
         # Attributes related to the data and the model
         self.val_data = (dataset.x_val, dataset.y_val)
         self.test_data = (dataset.x_test, dataset.y_test)
+        self.dataset_name = dataset.name
         self.generate_new_model = dataset.generate_new_model
+        self.init_model_from = init_model_from
+        self.use_weights_from_previous_coalition = use_weights_from_previous_coalition
```

Comment:
maybe rename to something like `use_saved_weights`. Or do we assume these weights come from a previous MPL experiment and not from elsewhere in the internet ?